### PR TITLE
ISLANDORA-2054: fix more than one field in SORT CRITERIA of a Views

### PR DESCRIPTION
**Pull request by giancarlobi**
**JIRA Ticket**: [ISLANDORA-2054](https://jira.duraspace.org/browse/ISLANDORA-2054)

* Other Relevant Links: https://groups.google.com/forum/#!topic/islandora/AXH0wm27kH0

# What does this Pull Request do?

When more than one field in SORT CRITERIA of a Views, the query string is wrong:
... &sort=dc.date_dt+asc*&*sort=fgs_label_s+asc& ...
instead of
... &sort=dc.date_dt+asc*,*fgs_label_s+asc& ...

# What's new?
No new features

# How should this be tested?

Build a views with more then one field in SORT CRITERIA.

# Additional Notes:
No additional information.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/7-x-1-x-committers

### CODE REVIEW

**Comment by DiegoPino** [comment]
@giancarlobi thanks for this!

There are some tiny issues we need before getting this merged

1.- Travis is failing because of Drupal coding standards. According to Travis https://travis-ci.org/Islandora/islandora_solr_views/jobs/272370318#L706 there is a missing space after the comma
2.- If you could make the pull request comment/description match the format, basically moving the description of the fix after "What does this Pull Request do?", adding the JIRA ticket and URL to the JIRA to the beginning and starting the title with ISLANDORA-2054: etc. I know this all sounds picky (sorry) but helps JIRA make the connection with the pull, basically making this one appear in the JIRA ticket itself. Pluse helps people not following the list understand better the issue and review.

You can use this https://github.com/Islandora/islandora_solr_views/pull/39 as example for the format if you want of any other open pull request

Highly appreciate this changes and will merge once Travis passes. Thanks again!

**Comment by giancarlobi** [comment]
I hope I made well homeworks ...
I changed code in PR and modified title and content, is it right?
Thanks a lot @DiegoPino to be so patient !

**Review by DiegoPino — APPROVED** [review]

**Comment by DiegoPino** [comment]
Gracias @giancarlobi!! Merging now =)

**Pull request merged**
